src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/service/BookingHistoryService.kt

```diff
@@ -25,7 +25,8 @@ class BookingHistoryService(private val bookingHistoryRepository: BookingHistory
       probationMeetingType = booking.probationMeetingType.takeUnless { booking.isCourtBooking() },
       videoUrl = booking.videoUrl,
       comments = booking.comments,
-      createdBy = booking.createdBy,
+      createdBy = booking.amendedBy ?: booking.createdBy,
+      createdTime = booking.amendedTime ?: booking.createdTime,
     ).apply {
```

Comment:
Does this address the other issue which seemed to leave the old booking comments in place when recording the history of an amendment?

Reply:
Hmm didn't notice an issue there.  Will have to revisit.